pyazul/index.py

```diff
@@ -30,51 +43,72 @@ def __init__(self, auth1, auth2, certificate_path, environment='dev'):
             self.url = 'https://pagos.azul.com.do/webservices/JSON/Default.aspx'
             self.ALT_URL = 'https://contpagos.azul.com.do/Webservices/JSON/default.aspx'
 
+    def __issue_http_requests(
```

Comment:
I have my reservations with name mangling. I'm not sure if this could be confusing for some (as dunders commonly are). Is there something we're trying to avoid while using the dunders on this method?

Reply:
yes, we don't want the user to be able to call this method but rather use the `azul_request` method, and since python doesn't have "private methods" this is the way to go.